go/cmd/vtbackup/vtbackup.go

```diff
@@ -138,19 +139,37 @@ func registerFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&mysqlTimeout, "mysql_timeout", mysqlTimeout, "how long to wait for mysqld startup")
 	fs.StringVar(&initDBSQLFile, "init_db_sql_file", initDBSQLFile, "path to .sql file to run after mysql_install_db")
 	fs.BoolVar(&detachedMode, "detach", detachedMode, "detached mode - run backups detached from the terminal")
+	fs.DurationVar(&keepAliveTimeout, "keep-alive-timeout", keepAliveTimeout, "Wait until timeout elapses after a successful backup before shutting down.")
```

Comment:
We can reuse the flag from servenv called `lameduck-period` which specifies the duration to wait for after SIGTERM is received.
The flag can be registered with a call `servenv.RegisterFlags()`

I don't if we should use that flag instead or keep this `keep-alive-timeout`. cc @deepthi

Reply:
@GuptaManan100 I tried that but it doesn't work the way I wanted it to. Lameduck period kicks in after the HTTP server is no longer serving Prometheus `/metrics`. The purpose of `keep-alive-timeout` is to keep the HTTP server alive long enough for one more scrape interval occur, so we don't miss any metrics.

Reply:
@maxenglander That makes sense.